pom.xml

```diff
@@ -136,6 +137,11 @@
                 <artifactId>caf-api</artifactId>
                 <version>1.17.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.cafapi</groupId>
+                <artifactId>caf-logging</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
```

Comment:
This has been released now, and note that the Maven coordinates have changed.  It should now be:

    <groupId>com.github.cafapi.logging</groupId>
    <artifactId>caf-logging-logback</artifactId>
    <version>1.0.0-36</version>